<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorListDialog</class>
 <widget class="QDialog" name="AuthorListDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User identification - please select or create a user</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="AuthorListView"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="RemoveUserBtn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Remove user</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ListsLabelName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>List</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="AddNewBtn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Add new</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="OkBtn">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AuthorListView</tabstop>
  <tabstop>OkBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
